icons/repeat-2.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <polyline points="2 9 5 6 8 9" />
+  <path d="M13 18H7c-1.1 0-2-.9-2-2V6" />
+  <polyline points="22 15 19 18 16 15" />
+  <path d="M11 6h6c1.1 0 2 .9 2 2v10" />
```

Comment:
Some minor optimization, I'll apply it and merge, since otherwise LGTM.
```suggestion
  <path d="m2 9 3-3 3 3" />
  <path d="M13 18H7a2 2 0 0 1-2-2V6" />
  <path d="m22 15-3 3-3-3" />
  <path d="M11 6h6a2 2 0 0 1 2 2v10" />
```